projects/vdk-control-cli/src/vdk/internal/control/command_groups/job/create.py

```diff
@@ -63,15 +69,45 @@ def __create_cloud_job(self, team: str, name: str) -> None:
             f"Data Job with name {name} created and registered in cloud runtime by Control Service."
         )
 
-    def __create_local_job(self, team: str, name: str, job_path: str) -> None:
-        sample_job = self.__vdk_config.sample_job_directory
+    def __create_local_job(
+        self, team: str, name: str, job_path: str, sample_job_directory
+    ) -> None:
+        sample_job = self.__create_sample_job_dir(sample_job_directory)
         log.debug(f"Create sample job from directory: {sample_job} into {job_path}")
         cli_utils.copy_directory(sample_job, job_path)
         local_config = JobConfig(job_path)
         if not local_config.set_team_if_exists(team):
             log.warning(f"Failed to write Data Job team {team} in config.ini.")
         log.info(f"Data Job with name {name} created locally in {job_path}.")
 
+    def __create_sample_job_dir(self, sample_job_module):
+        """
+        This method generates the directory for the sample job which will be created when invoking `vdk create`.
```

Comment:
Where does the method actually "create/generate" the dir?

Reply:
resolving this since it was merged in a separate PR